### ISSUE TITLE
[question]Loading Pre-trained flow model.

### ISSUE THREAD

**Issue by ksammoh** [gsig/charades-algorithms#13]
Hi,

 I came across your data set and repo while researching about video analysis.
 Thank you very much for sharing your rich data set and code.

 I'm using pytorch implementation and able to load the rgb model without any problem.
 But I', facing problems while loading flow model downloaded from https://www.dropbox.com/s/m1hkeiwjtndt26z/twostream_flow.pth?dl=1. 

Specifically, I face error @ # replace last layer block of code in ```__init__.py``` file in the final else case.
System says "Dropout does not have field in_features..."

Could you please give me some direction on how to overcome this issue?
Thanks in advance!

**Comment by gsig** [comment]
Seems like I introduced code that didn't support the flow model at some point. Should be fixed now in the latest commit. Let me know if it works.

**Comment by ksammoh** [comment]
Thank you very much Gunnar.
I tested your changes and they are working fine.

Thanks a lot for quick reply!
